environments/icds-cas/app-processes.yml

```diff
@@ -693,6 +693,8 @@ pillows:
       start_process: 117
       num_processes: 3
       total_processes: 120
+    case_messaging_sync_pillow:
```

Comment:
I'm confused. this isn't showing up under changes to process by pillow https://app.datadoghq.com/dashboard/ewu-jyr-udt/change-feeds-pillows?from_ts=1565652360869&to_ts=1565738760869&live=true&tile_size=m&tpl_var_env=icds&fullscreen_widget=186199249&fullscreen_section=overview

Reply:
I think it's because I haven't updated localsettings to it's not listed in `ACTIVE_PILLOW_NAMES`. I'll do that today